src/mtcp/mtcp_restart.c

```diff
@@ -184,11 +184,11 @@ MTCP_PRINTF("Attach for debugging.");
 {int x=1; while(x);}
 #endif
 
-// TODO(karya0): Remove vDSO checks after 2.4.0-rc3 release.
+// TODO(karya0): Remove vDSO checks after 2.4.0-rc3 release, and after testing.
 // Without mtcp_check_vdso, CentOS 7 fails on dmtcp3, dmtcp5, others.
 #define ENABLE_VDSO_CHECK
 // TODO(karya0): Remove this block and the corresponding file after sufficient
-// testing.
+// testing:  including testing for __i386__, __arm__ and __aarch64__
 #ifdef ENABLE_VDSO_CHECK
```

Comment:
But I thought we were keeping `#ifdef ENABLE_VDSO_CHECK` along with `#define ENABLE_VDSO_CHECK` because otherwise MVAPICH fails on Stampede (in 64-bit mode).  This was the essence of this comment:  https://github.com/dmtcp/dmtcp/pull/60#issuecomment-94137963 .  That pull request had been closed when we added  `#define ENABLE_VDSO_CHECK` to the master branch.